Modules/CustomRolesHelper.cs

```diff
@@ -883,6 +846,10 @@ CustomRoles.Witness or
             CustomRoles.Monarch or
             CustomRoles.Farseer or
             CustomRoles.SwordsMan or
+            CustomRoles.Admirer or
+            CustomRoles.Divinator or

```

Comment:
How are Fortune Teller (Divinator) and Merchant taskless? lol

Reply:
They're based on the kill button

Reply:
They both use a crewmate basis.
Merchant's ability uses tasks, while Fortune Teller uses votes (and has tasks).

Reply:
Hhhh, i will fixed